### PR TITLE
Set up Travis CI for CI/CD pipeline

### PR DESCRIPTION
**Pull request by blakes24**
### What does this PR do?
- This PR adds the .yml file with the configurations to set up Travis CI for the CI/CD pipeline

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  include:
+    - language: python
+      python: 3.7
+      install:
+        - pip install -r requirements.txt
+      script:
+        - python -m unittest
+
+    - language: node_js
+      node_js: "node"
```
Comment (alexkayabula) [inline .travis.yml]: @blakes24 I suspect that the build could be failing due to the node_js version specified. Remove "node".

Comment (alexkayabula) [reply .travis.yml]: @blakes24 do you still need more assistance on this pull request ? If not kindly update the badges.

**Comment by blakes24** [comment]
The tests passed but it says the build configuration language is set to Ruby. Is there another setting I need to change?

**Comment by alexkayabula** [comment]
> The tests passed but it says the build configuration language is set to Ruby. Is there another setting I need to change?

Let me checkout the build logs and confirm.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,12 @@
+matrix:
+  include:
+    - language: python
+      python: 3.7
```
Comment (alexkayabula) [inline .travis.yml]: @blakes24 update the syntax to ```python: "3.7"```

**Comment by blakes24** [comment]
Tests pass and build language is set to python.

**Pull request merged**